testsuite/testcases/src/performance_test.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::generate_traffic;
+use forge::{NetworkContext, NetworkTest, Result, Test};
+use tokio::time::Duration;
+
+pub struct PerformanceBenchmark;
+
+impl Test for PerformanceBenchmark {
+    fn name(&self) -> &'static str {
+        "performance::performance-benchmark"
+    }
+}
+
+impl NetworkTest for PerformanceBenchmark {
+    fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
+        let duration = Duration::from_secs(240);
+        let all_validators = ctx
+            .swarm()
+            .validators()
+            .map(|v| v.peer_id())
+            .collect::<Vec<_>>();
+
+        // Generate some traffic
+        let _txn_stat = generate_traffic(ctx, &all_validators, duration)?;
```

Comment:
I would expect that we'd output the stats.

Although I don't particularly like the fact that dips in perf can fail the existing cluster test, in order to have feature parity we should probably support that. Perhaps by:

```rust
pub struct PerformanceBenchmark {
    tps_threshold: Option<u64>,
}
```

Where if tps_threshold is Some(x) that we fail if tps is lower that x

Reply:
I will add threshold to indicate performance regression

Reply:
As we discussed offline, perf fail detection will be added in next PR for ci-test change

Reply:
Do we want to output the stats?